packages/web/src/_page-tests/__snapshots__/about.test.tsx.snap

```diff
@@ -724,7 +724,7 @@ exports[`About renders 1`] = `
             />
           </audio>
         </div>
-        
```

Comment:
someday id like to figure out why there are these whitespace changes sometimes on this files

Reply:
I think it's my VS Code config that automatically delete trailing whitespace (which we really should have everywhere)